passes/lang0.md

```diff
@@ -100,5 +100,5 @@ top-level node, in dedicated sections, to allow for easy and fast access to
 them.
 
 ```grammar
-module ::= (Module (TypeDefs <type>*) (GlobalDefs <type_id>*) (ProcDefs <procdef>*))
+top ::= (Module (TypeDefs <type>*) (GlobalDefs <type_id>*) (ProcDefs <procdef>*))
```

Comment:
```suggestion
module ::= (Module (TypeDefs <type>*) (GlobalDefs <type_id>*) (ProcDefs <procdef>*))
top ::= <module>
```

Any reason not to go this route like in `lang_source`, might be slightly more self-documenting. This isn't a blocker, feel free to ignore.

Reply:
Yea, that's better.